M2/Macaulay2/bin/Makefile.in

```diff
@@ -115,7 +115,7 @@ endif
 
 # the order of this list should agree with that in ../../configure.ac
 M2_LIBRARIES :=
-M2_LIBRARIES +=  @BUILTLIBS@ @LINALGLIBS@ @LIBS@ @FCLIBS@
+M2_LIBRARIES +=  @BUILTLIBS@ @LINALGLIBS@ @LIBS@ @FCLIBS@ -lboost_regex
```

Comment:
I'm not sure if there's a more standard way to link boost_regex with autotools.

Reply:
One possibility would be [ax_boost_regex](https://www.gnu.org/software/autoconf-archive/ax_boost_regex.html) from the Autoconf Archive.

Reply:
Feel free to push a commit if you're interested and have time!

Reply:
Done!

Reply:
Thank you!